src/flick/suggestion/views.py

```diff
@@ -0,0 +1,61 @@
+import json
+from user.models import Profile
+
+from api import settings as api_settings
+from api.utils import failure_response
+from api.utils import success_response
+from django.contrib.auth.models import User
+from friendship.models import Friend
+from rest_framework import generics
+from show.models import Show
+
+from .models import PrivateSuggestion
+from .serializers import PrivateSuggestionSerializer
+
+
+class PrivateSuggesionView(generics.GenericAPIView):
+    permission_classes = api_settings.CONSUMER_PERMISSIONS
+
+    def get(self, request):
+        user = request.user
+        suggestions = user.suggestions_received.all()
+        suggestion_data = PrivateSuggestionSerializer(suggestions, many=True)
+        return success_response(suggestion_data.data)
+
+    def post(self, request):
+        data = json.loads(request.body)
+        user = request.user
+        if not Profile.objects.filter(user=user):
+            return failure_response(f"{user} must be logged in.")
+        profile = Profile.objects.get(user=user)
+
+        show_id = data.get("show_id")
```

Comment:
lets do `show_id = data.get("show")` to be consistent with our list create and list update